test/examples/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:16-alpine
+
+USER node
+ENV TZ=UTC
+RUN mkdir /home/node/app
+WORKDIR /home/node/app
+
+COPY --chown=node:node ["package*.json", "*.lock", "./"]
+RUN npm install
```

Comment:
`npm ci`?

Reply:
No package-lock in the examples => no `npm ci`.

Reply:
That lock file is for the test runner. I'm using the same Dockerfile for the examples as well as the test runner.

To use `npm ci` for the test runner(not a lot of benefits since there's like 1 dep) I should separate Dockerfiles and have a separate one for the test runner - not worth it imho.